src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -640,7 +640,16 @@ namespace ZoneEquipmentManager {
 				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).HeatMassFlow = 0.0;
 				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).HeatZoneTemp = 0.0;
 				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).HeatZoneHumRat = 0.0;
-			}
+			} else {
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolLoad = 0.0;
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolMassFlow = 0.0;
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolZoneTemp = Node( ZoneNode ).Temp;
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolZoneHumRat = ZoneSizingInput( ControlledZoneNum ).CoolDesHumRat;
```

Comment:
@Nigusse @rraustad I suspect this may be why the unit test fails on some platforms, but I have not confirmed that.